python/imageutils.py

```
@@ -12,9 +12,7 @@
 
 
 class Merge(Enum):
-    block_block = 0
-    block_slice = 1
-    slice_slice = 2
+    clustered = 0
```

unclustered could also be triggered with memory = 0

---

or should mem of 0 throw an error? right now it's unclustered if mem is not provided or mem = some number of bytes in which only one block can fit....and will do absolutely nothing if mem is 0 -which is a problem!

---

I'd say mem=0 should be unclustered then.